src/components/results/candidate-results/personality-dimension-details/details-list/index.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from "prop-types";
+import {rgba} from "lib/helpers/color";
+import style from "../style";
+
+export default function DetailsList(props) {
+  const {color, detailsList, header} = props;
+
+  return (
+    <div>
+      <h3>{header}</h3>
+      <ul className={style.detailsGroup}>
+        {detailsList.map((listItem) => (
+          <li key={listItem} style={{background: rgba(color, 50)}}>{detailsList}</li>
```

Comment:
Looks like the tests are failing because `detailsList` instead of `listItem` here. Also for the variables, I think you can just use something like `details` and `detail` or `item` since the list part is pretty implied